src/components/Node.tsx

```diff
@@ -86,13 +87,15 @@ function InlineEditor({
   onStopEditing,
 }: InlineEditorProps): JSX.Element {
   const [node] = useNode();
+  const { setIsInputElementInFocus } = useInputElementFocus();
```

Comment:
instead of doing this everywhere we use Quill, I suggest to write a wrapper as you did with the other FormControls

Reply:
unresolved

Reply:
done